### PR TITLE
proposal: move PROJECT_ONBOARDING to stage 3

### PR DESCRIPTION
**Pull request by MylesBorins**
This needs to land with #165

Refs: https://github.com/openjs-foundation/cross-project-council/pull/165

### CODE REVIEW

**Comment by MylesBorins** [comment]
@mkdolan @brianwarner can you please approve this once the board has been consulted and +1s

File: proposals/README.md [hunk 1]
```diff
@@ -24,4 +24,5 @@ Anyone may submit an idea for a policy or program following the [staging process
 * [Expectations](./stage-3/EXPECTATIONS)
 * [Governance](./stage-3/GOVERNANCE)
 * [Individual Membership program](./stage-3/INDIVIDUAL_MEMBERSHIP)
+* [Project Onboarding](./stage-2/PROJECT_ONBOARDING)
```
Comment (kborchers) [inline proposals/README.md]: Link should be to stage-3

**Comment by kborchers** [comment]
@MylesBorins as CPC director, should I take this to the board for review or would you prefer we leave this with @brianwarner and @mkdolan?

**Comment by MylesBorins** [comment]
@kborchers it would likely make sense for you to pick this one up... happy to run with it thought

**Comment by kborchers** [comment]
The board has no objections to this process so I believe it is ready to move to stage 3 unless there are any other objections or feedback from the CPC

**Comment by mhdawson** [comment]
@MylesBorins can you rebase and then we'll land.

**Comment by mhdawson** [comment]
Discussed in today's meeting, no additional objections raised so landing.

**Comment by mhdawson** [comment]
Checklist for PR's changing governance which I think this falls under:

As per: https://github.com/openjs-foundation/cross-project-council/blob/master/GOVERNANCE.md
-[X] no outstanding objections
-[X] two or move approvals from CPC members
-[X] the PR has been open for at least 14 days

Since this is a significant part of what will be the CPC governance
-[X] FYI to board and no objections from the board.

**Comment by mhdawson** [comment]
From the comment in the first commit, looks like this also needs to land with #165, probably better to wait for @MylesBorins to come back to make sure we're covering what is needed from #165 as well.

I've updated to resolve the conflict.

**Pull request merged**